bears/jinja2/Jinja2Bear.py

```diff
@@ -5,6 +5,24 @@
 from coalib.results.Diff import Diff
 
 
+def generate_indent_diff(file, filename, line, line_number, indent_level):
+    """
+    Generate a diff for incorrectly indented line.
+
+    :param indent_level: The number of spaces the is expected at the start
```

Comment:
Why only one param?

Reply:
To maintain consistency with the rest of the code. The other param haven't been documented in the below functions.